tools/automatic_query_fixer/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group 'com.google.cloud.bigquery.utils'
+version '1.0-SNAPSHOT'
+
+
+
+application {
+    mainClassName = "com.google.cloud.bigquery.utils.auto_query_fixer.Application"
+}
+
+repositories {
+    mavenCentral()
+    flatDir {
+        dirs 'libs'
+    }
+
+}
+
+dependencies {
+
+    compileOnly 'org.projectlombok:lombok:1.18.12'
+    annotationProcessor 'org.projectlombok:lombok:1.18.12'
+    implementation 'org.apache.calcite.avatica:avatica-core:1.17.0'
```

Comment:
We still decided to use this :)?

Reply:
Good point. It seems some place can be replaced by `AutoValue`, but not every class is acted as an immutable value and some may need to be modified (by setter) after creation. Thus, I think `lombok` may be better in some situations. If `AutoValue` can do the similar job, I will change to it.

Reply:
Autovalue builder pattern can help you generate getters/setters. 
https://github.com/google/auto/blob/master/value/userguide/builders.md
We are open to use lib we want not sure if one is significant better than the other given the current use case.